androidtv/constants.py

```diff
@@ -28,20 +30,30 @@
 #: Determine whether the device is awake
 CMD_AWAKE = "dumpsys power | grep mWakefulness | grep -q Awake"
 
+#: Define current app
+CMD_DEFINE_CURRENT_APP = "CURRENT_APP=$(dumpsys window windows | grep -E 'mCurrentFocus|mFocusedApp') && " + VAR_CURRENT_APP_EXTRACTION
+
 #: Get the current app
-CMD_CURRENT_APP = "CURRENT_APP=$(dumpsys window windows | grep mCurrentFocus) && CURRENT_APP=${CURRENT_APP#*{* * } && CURRENT_APP=${CURRENT_APP%%/*} && echo $CURRENT_APP"
+CMD_CURRENT_APP = CMD_DEFINE_CURRENT_APP + ' && echo $CURRENT_APP'
+
+#: Define current app for a Google TV device
+CMD_DEFINE_CURRENT_APP_GOOGLE_TV = 'CURRENT_APP=$(dumpsys activity a . | grep mResumedActivity) && ' + VAR_CURRENT_APP_EXTRACTION
 
 #: Get the current app for a Google TV device
-CMD_CURRENT_APP_GOOGLE_TV = "CURRENT_APP=$(dumpsys activity a . | grep -E 'mResumedActivity' | cut -d ' ' -f 8) && CURRENT_APP=${CURRENT_APP%%/*} && echo $CURRENT_APP"
+CMD_CURRENT_APP_GOOGLE_TV = CMD_DEFINE_CURRENT_APP_GOOGLE_TV + ' && echo $CURRENT_APP'
 
 #: Get the HDMI input
-CMD_HDMI_INPUT = "dumpsys activity starter | grep -o 'HDMIInputService\\/HW[0-9]' -m 1 | grep -o 'HW[0-9]'"
+CMD_HDMI_INPUT_SEARCH = r" | grep -E -o '(ExternalTv|HDMI)InputService/HW[0-9]' -m 1 | grep -o 'HW[0-9]'"
```

Comment:
I'd rather not have a constant that's only used once.

Reply:
Again, it seems Chrome on iOS failed me. I see that you also used it in a unit test. 

Still, I don't think it's necessary to have this constant. Your unit test could simply strip off "dumpsys activity starter" from the command.

Reply:
Nice suggestion! Instead of stripping `dumpsys activity starter`, I decided to replace `dumpsys activity starter` with the dumpsys output as specified in test case.